web-packages/imgui-demo/src/panels/editor-window.ts

```
@@ -21,7 +22,7 @@ export class EditorWindow {
       this.firstFrame = false;
     }
 
-    ImGui.Begin(this.title, (value = this.opened) => this.opened = value);
+    ImGui.Begin(this.title, (value = this.opened) => this.opened = value, this.windowFlags);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)